package.json

```diff
@@ -66,10 +67,8 @@
     "webpack-dev-server": "^4.15.1"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": [
-      "eslint --cache --fix",
-      "prettier --cache --write --ignore-unknown --list-different"
-    ]
+    "packages/**/src/**/*.{ts,tsx}": "eslint --cache --fix --max-warnings 0",
```

Comment:
I have some reservations about this change. This change affects the whole patient chart mono repo and I see you excluded `js,jsx`. While we're mostly using typescript there might be some js files

cc: @denniskigen

Reply:
I removed changes in the lock file